packages/opentelemetry-exporter-zipkin/src/types.ts

```diff
@@ -187,3 +188,15 @@ export type SendFunction = (
 ) => void;
 
 export type GetHeaders = () => Record<string, string> | undefined;
+
+/**
+ * Represents a timed event.
+ * A timed event is an event with a timestamp.
+ */
+export interface TimedEvent {
```

Comment:
Isn't it possible to reuse the `TimedEvent` type from trace module here?

Reply:
well I wasn't really sure what is better, expose TimedEvent from trace or not. I don't have really strong opinion if this should be exposed and we should reuse or wee should expose and not reuse or we should rather treat this interface as kind of private and not expose at all. Based on that I simply created the missing event to avoid the dependency and leave us some freedom. Butt I'm fine with any of the solution.

Reply:
Just because zipkin's timed event has the same properties as ours isn't related. I think they are separate types that just happen to look the same.

Reply:
I think it's needed that trace exports it as part of a `ReadableSpan` otherwise exporters can't export it.

Reply:
so then it stays as it is, are you ok with this @Flarna ?

Reply:
I'm not sure what i was thinking on my earlier comment. The trace module should export the type so it can be used. If the trace module exports readable span, it should also export every type needed to properly reference it.

Reply:
It is implicit exported already by trace module via `typeof ReadableSpan['events']`.
Clearly it would be easier to reuse by explicit exporting it there.

So my vote is for explicit export in trace and reuse.

Reply:
but timed event is already exported from trace

Reply:
So why are we not using it?